lib/activerecord_slotted_counters/adapters/rails_upsert.rb

```diff
@@ -3,7 +3,7 @@
 module ActiveRecordSlottedCounters
   module Adapters
     class RailsUpsert
-      attr_reader :klass
+      attr_reader :klass, :current_adapter_name
```

Comment:
Where do we use `current_adapter_name`?

Reply:
Yep, sorry, It was leaked from another branch
fixed